### ISSUE TITLE
Dosnt work after Release of "Seekers of the Storm"-DLC

### ISSUE THREAD

**Issue by KotonaGH** [SSM240/FadeEmptyChests#3]
The mod is marked as "Deprecated" in r2modman.
The Main Game got an update with the Release of the DLC wich broke the mod.
Please update the mod to the newest Version.

Thank you

**Comment by SSM240** [comment]
Yes I'm aware, there's a reason I marked it as deprecated lol. Unfortunately I'm not even sure there's much I can do from my side, since the core feature it relies on (chests being able to become transparent) is straight-up broken in the new version. The code for it is all still there and should work, it just doesn't for some reason lol.
